packages/v4/src/content/design-guidelines/components/select/select.md

```diff
@@ -9,7 +9,29 @@ related: [
 ]
 ---
 
-**Select lists** enable users to select one or more items from a list. Select lists should also be used when options are dynamic or variable. 
+**Select lists** enable users to select one or more items from a list. Select lists should also be used when options are dynamic or variable.
+
+## Elements and variations
+
+### Basic select
+A select component is a type of dropdown menu that consists of a toggle control to open and close a menu of actions or links. Selects differ from dropdowns in that they persist selection where dropdowns are typically used to present a list of actions or links.
+
+<img src="./img/basic-select.png" alt="basic select menu" width="303"/>
+
+1. **Toggle:** The toggle is used to open and close the menu. It takes on all of the behaviors of a button. The toggle label always reflects the currently selected item or may hold a placeholder if nothing is selected.
+
+2. **Menu:** The menu contains a list of options or values. Menus should optimally contain seven (7) or less items for easy scanning by the user. For longer menus, items may be grouped. See the [Menu component guidelines](/components/menu/design-guidelines) for more information about supported options, including use of icons, grouping, and descriptions. The Dropdown component includes many common built-in menu variants. It’s also possible to attach a custom menu by using the [Basic panel variant](/components/select/react#panel).
```

Comment:
This is confusing - did you mean to change the second half of the text but forgot to? It seems like it's talking about drop downs even though this is the select guidelines?
I also would want to add a section on scrolling here, or some other recommendations for when the select list is too long. In most case, if someones list is too long, grouping it wouldn't help with that issue! I'd expect a recommendation of making the menu scrollable, or adding a "see more" button at the bottom to view more options. I know this is a common battle/question in c.rh.c

Reply:
Yes, this is confusing. I did a quick copy and paste without reading this thoroughly. Hopefully it's better now.